QueueImplementation/VM/CustomCollectionViewModel.cs

```diff
@@ -0,0 +1,363 @@
+﻿using Business.Helper;
+using Business.Interfaces;
+using MvvmCross.ViewModels;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+
+namespace ViewModel
+{
+	/// <summary>
+	/// Represents the view model used for managing the collection.
+	/// </summary>
+	/// <owner>Anton Petrenko</owner>
+	public sealed class CustomCollectionViewModel<T> : MvxViewModel
```

Comment:
Why do we need the type if we don't use it?
Lets discuss.

Reply:
Removed generic parameter.